lib/google-civic/client.rb

```diff
@@ -11,38 +11,44 @@ def initialize(options={})
     include GoogleCivic::Request
 
     # Gets a list of elections in the API
-    #
+    # @param options [Hash] A customizable set of options for the request. Any options for the
+    # connection should be nested inside under the key `:connection`.
     # @return [Hashie::Mash] A list of current elections in the API
     # @see https://developers.google.com/civic-information/docs/us_v1/elections/electionQuery
     # @example List the current elections
     #   GoogleCivic.elections
     def elections(options={})
-      get("elections", options)
+      connection_options = options.delete(:connection) {|el| {}}
```

Comment:
Yeah, I mostly forgot what the "ignore a parameter" syntax was in Ruby for a while, I think I figured it out later.